manual/custom_codecs/extras/README.md

```diff
@@ -0,0 +1,417 @@
+## Optional codecs
+
+The driver's "extras" module provides additional [codec](../) implementations: these codecs are not required by core
+driver features, but will probably prove useful in a lot of client applications. You can also study their source code as
+a reference to write your own.
+
+The module is published as a separate Maven artifact:
+
+```xml
+<dependency>
+  <groupId>com.datastax.cassandra</groupId>
+  <artifactId>cassandra-driver-extras</artifactId>
+  <version>3.0.0-rc1</version>
+</dependency>
+```
+
+It also declares a number of optional dependencies, that you'll need to explicitly re-declare in your application
+depending on which codecs you plan to use (this will be covered for each case below).
+
+### Time types
```

Comment:
I would suggest the term "temporal" to avoid any confusion with the CQL `TIME` type.